lib/4_filter_numbers.js

```diff
@@ -6,6 +6,10 @@
 
 module.exports = {
   filter_numbers: function(arr){
-    // Your code goes here.
+  function lessthanten(number) {
```

Comment:
I like your instinct to split these up into two functions. Can you say more about why you chose to do this?